v3/lints/cabf_br/lint_sub_ca_eku_incompatible_values_test.go

```diff
@@ -0,0 +1,67 @@
+package cabf_br
+
+/*
+ * ZLint Copyright 2021 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"testing"
+
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/test"
+)
+
```

Comment:
It probably makes sense to switch this to a so-called "table based test" - e.g. https://github.com/zmap/zlint/blob/48b300e877f2ed3e6b7e75426a273f84455f17a6/v3/lints/mozilla/lint_e_prohibit_dsa_usage_test.go#L24

Eliminates a lot of redundancy, which in this case, every body is more or less the same other than the filename and expected result.